test/unit/metrics/test_plot_metricframe.py

```diff
@@ -0,0 +1,184 @@
+    """Return standard error (for binary classification).
+
+    Assumes infinitely large population.
+    Should be used when the sampling fraction is small.
+    For sampling fraction > 5%, may want to use finite population correction [1]
+
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Margin_of_error
+    """
+    return round(z*np.sqrt(p*(1.0-p))/np.sqrt(n), digits)
+
+
+def recall_wilson(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    tn, fp, fn, tp = confusion_matrix(y_true, y_pred).ravel()
+    bounds = general_wilson(tp/(tp+fn), tp + fn, digits_of_precision, z_score)
+    return bounds
+
+
+def incorrect_flipped_recall_wilson(y_true, y_pred):
+    bounds = recall_wilson(y_true, y_pred)
+    return bounds[1], bounds[0]
+
+
+def incorrect_single_bound_recall_wilson(y_true, y_pred):
+    bounds = recall_wilson(y_true, y_pred)
+    return bounds[0]
+
+
+def recall_normal_err(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    tn, fp, fn, tp = confusion_matrix(y_true, y_pred).ravel()
+    error = general_normal_err_binomial(tp/(tp+fn), tp + fn, digits_of_precision, z_score)
+    return error
+
+
+def incorrect_negative_recall_normal_err(y_true, y_pred):
+    error = recall_normal_err(y_true, y_pred)
+    return -1 * error
+
+
+def accuracy_wilson(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    score = accuracy_score(y_true, y_pred)
+    bounds = general_wilson(score, len(y_true), digits_of_precision, z_score)
+    return bounds
+
+
+def accuracy_normal_err(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    score = accuracy_score(y_true, y_pred)
+    error = general_normal_err_binomial(score, len(y_true), digits_of_precision, z_score)
+    return error
+
+
+metrics_dict = {
+    'Recall': recall_score,
+    'Recall Bounds': recall_wilson,
+    'Recall Error': recall_normal_err,
+    'Accuracy': accuracy_score,
+    'Accuracy Bounds': accuracy_wilson,
+    'Accuracy Error': accuracy_normal_err,
+    'Recall Bounds Flipped': incorrect_flipped_recall_wilson,
+    'Recall Bounds Single': incorrect_single_bound_recall_wilson,
+    'Recall Error Negative': incorrect_negative_recall_normal_err,
+}
+
+
+@pytest.fixture()
+def sample_metric_frame():
+    return MetricFrame(
+        metrics=metrics_dict,
+        y_true=y_t,
+        y_pred=y_p,
+        sensitive_features=g_1)
+
+
+def test_plotting_output(sample_metric_frame):
+    axs = plot_metric_frame(sample_metric_frame).flatten()
+    assert len(axs) == 6  # 6 is number of metrics that aren't arrays
+    assert isinstance(axs[0], matplotlib.axes.Axes)
+
+
+def test_ambiguous_error_metric(sample_metric_frame):
+    with pytest.raises(ValueError) as exc:
+        plot_metric_frame(sample_metric_frame, metrics=["Recall"], errors=[
+                          "Recall Error"], conf_intervals=["Recall Bounds"])
+        assert str(exc.value) == _GIVEN_BOTH_ERRORS_AND_CONF_INT_ERROR
```

Comment:
I'm not sure if this line ever runs since the previous line is supposed to raise

Reply:
Good point - I think that it needs to be unindented. You can check @alexquach  by comparing to some random string, and seeing if the test fails.

Reply:
good catch, unindenting it made it work